project/views.py

```diff
@@ -79,3 +79,11 @@ def humans_of_xr(request, id=None):
         return render(request, 'human.html', {'human': Human.objects.get(id=id)})
     else:
         return render(request, 'humans-of-xr.html', {'humans': Human.objects.filter()})
+
+def group_detail(request, id):
+    group = Group.objects.get(id=id)
```

Comment:
If we use get_object_or_404 as in Event and Article views then we won't get 500 errors when there is a bad link or someone guessing a page.